src/main/java/me/totalfreedom/totalfreedommod/command/Command_expel.java

```diff
@@ -82,7 +82,7 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         }
         else
         {
-            msg("Pushed " + pushedPlayers.size() + " players: " + StringUtils.join(pushedPlayers, ", "));
+            msg("Pushed " + pushedPlayers.size() + " players: " + StringUtils.join(pushedPlayers, ", ") + ".");
```

Comment:
Think this is another one of those adding an extra . to the end of names that I'm not keen on...

Reply:
done